ci_environment/mysql/templates/default/my.cnf.erb

```diff
@@ -51,14 +51,17 @@ basedir         = /usr
 datadir         = <%= node['mysql']['data_dir'] %>
 tmpdir          = /tmp
 skip-external-locking
+
 #
 # Instead of skip-networking the default is now to listen only on
 # localhost which is more compatible and is not less secure.
 bind-address            = <%= node['mysql']['bind_address'] %>
 
+innodb_file_per_table  = true
```

Comment:
Is this necessary? http://dev.mysql.com/doc/refman/5.5/en/innodb-multiple-tablespaces.html lists advantages, but not many are relevant for throw-away test databases.